mailarchive/mailarchive-subetha/src/bundle/sakaimailet.properties

```diff
@@ -17,3 +17,5 @@ mail.support.683=Sender's email does not belong in Sakai, nor is it recognized a
 mail.support.521=Destination address of this e-mail couldn't be retrieved.  Please, reply this message from Sakai's Messages tool.
 
 mail.support.421=Your email could not be sent due to connectivity problems. Please, try again later.
+
+mail.support.358=Message body has characters that cannot be used. Please look over your email content and try again. You can also reply from Aula Virtual.
```

Comment:
@victorGomollon, is "Aula Virtual" an institution-specific site or technology?

Reply:
maybe use the institution service

Reply:
Sakai.property ui.service

Reply:
Resolved